### ISSUE TITLE
Extension collapses search bar in Ubuntu app view

### ISSUE THREAD

**Issue by rakshitraj** [mlutfy/hidetopbar#329]
When On, the extension will collapse the search bar in Ubuntu options
System: Ubuntu 21.10 LTS
Screenshots below:
![Screenshot from 2022-01-15 04-10-10](https://user-images.githubusercontent.com/31768189/149594777-860e762a-40fb-46ce-b216-f429f71fa07b.png)
Err: when the extension is on. Look at the search bar

![Screenshot from 2022-01-15 04-10-15](https://user-images.githubusercontent.com/31768189/149594784-934ac3dd-3c50-43cf-adda-2158a82714d0.png)
Typical behavior, when extension is disabled

Extension settings:
![Screenshot from 2022-01-15 04-10-51](https://user-images.githubusercontent.com/31768189/149594786-8b32f2e4-7f88-4734-bf68-dd4ad4dad26a.png)
![Screenshot from 2022-01-15 04-10-57](https://user-images.githubusercontent.com/31768189/149594788-a0a995a4-40a8-470c-be77-b2d8ac5fede0.png)

**Comment by tuxor1337** [comment]
* Which version of GNOME do you use?
* Which version of Hide Top Bar do you use?
* What other GNOME extensions are enabled in parallel?
* Does the problem also appear when all other GNOME extensions are disabled?

**Comment by tuxor1337** [comment]
Closing as duplicate of #321

**Comment by rakshitraj** [comment]
Can confirm the duplicate issue. Readers please refer to #321 since it is documented better.
